packages/react-pdf-toolkit/src/utils/sanitize-string.ts

```diff
@@ -0,0 +1,4 @@
+export const sanitizeString = (string: string): string => {
```

Comment:
Redundant return type.
```suggestion
export const sanitizeString = (string: string) => {
```

Reply:
P3